Payara-Micro/spring-boot-jpa-jsf-example/README.md

```diff
@@ -0,0 +1,11 @@
+Compile with:
+
+    mvn compile
```

Comment:
Wouldn't that be "mvn package" which comes after "compile" and before "install"?